hamilton/parameter_input/lhs_sampling.py

```diff
@@ -48,6 +69,8 @@
     help="Output file path, relative to current working directory",
 )
 def cli(num_param_sets, output_file):
+    check_parameter_dictionary()
+
     limits = []
     for param_range_data in PARAM_DATA:
         limits.append([param_range_data[P_START], param_range_data[P_END]])
```

Comment:
Here's where I mean. This could be something like:

```python
for param_name in VARIABLE_PARAM_NAMES:
   param_range_data = PARAM_DATA[param_name]
   limits.append(...)
```

One extra line here saves more complex checks higher up.